test-snapshots/snapshots-output/Snapshot.Constructor.ss

```diff
@@ -0,0 +1,47 @@
+#!r6rs
+#!chezscheme
+(library
+  (Snapshot.Constructor lib)
+  (export
+    Just*
+    Just-value0
+    Just?
```

Comment:
Something to consider for later (perhaps?) but does it matter to the Chez Scheme code that the `Just` type isn't exported?

Reply:
Tracked in https://github.com/purescm/purescm/issues/92